hobby_blog.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="styles/hobby_blog.css">
+  <link rel="stylesheet" href="normalize.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300"
+  rel="stylesheet">
+  <title>Chantal's Hobby Blog</title>
+</head>
+
+<body>
+  <img alt="foggy blue mountains" src="assets/background.jpg" />
+
+    <header>
+      <nav class="site-nav">
+        <a href="about_me.html">About Me</a>
+        <a href="portfolio.html">Portfolio</a>
+        <a href="hobby_blog.html">Hobby-Blog</a>
+        <a href="index.html">Home</a>
+      </nav>
+    </header>
+
+
+  <section id="hobbies">
```

Comment:
This could maybe be structured as follows:
```
<section id="hobbies">
    <article>
        <section class="hiking">
            Lots of interesting stuff
        </section>
        <section class="gaming">
            More interesting stuff
        </section>
    </article>
</section>
```
In the blog post below, I found the helpful analogy to be to think about it like a newspaper with different sections (sports, news, classifieds).

This type of thought could be applied anywhere there are sections, articles, divs... the reason this pieces of code in particular jumped out at me is because Hiking and Gaming have their own headers.

https://www.iandevlin.com/blog/2011/04/html5/html5-section-or-article